lib/events/Events.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+var _ = require("lodash");
+var Environment = require("../environment/Environment");
+var Backbone = require("../application/Backbone");
+var noop = require("../util/noop");
+
+var ServerEvents = _.extend({},
+    Backbone.Events, {
+        on: noop,
+        once: noop,
+        listenTo: noop,
+        listenToOnce: noop,
+        delegateEvents: function() {
```

Comment:
delegateEvents is not part of Backbone.Events, so user can't turn on server side event hash using following code:
```
var View = Brisket.View.extend(_.extend({},
    Backbone.Events, {
        // View code
    }
)
```